server/test/server.test.ts

```diff
@@ -0,0 +1,31 @@
+import mocha from 'mocha';
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+
+// TODO figure out why this doesn't work yet..
+//import 'chai/register-expect';
+
+import { app } from '../src/App';
+
+chai.use(chaiHttp);
+const expect = chai.expect;
+
+describe('baseRoute', () => {
+  it('/ should be static html', () => {
+    return chai
+      .request(app)
+      .get('/')
+      .then(res => {
+        expect(res.type).to.eql('text/html');
+      });
+  });
+
+  it('/ should have a welcome message', () => {
+    return chai
+      .request(app)
+      .get('/')
+      .then(res => {
+        expect(res.text).to.include('Welcome to Arcs');
```

Comment:
Do we expect this to be accessible directly? Actually, a couple of paragraphs about the scope of this (what will access what APIs) would be useful, could even just be the commit message then copy to a doc to flesh out later.

Reply:
Server landing page and APIs is TBD as we are still in an exploratory phase.

Also note that these tests will get wired into CI in a follow on patch.